bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -13,6 +13,12 @@ pub struct ComposerModel {
     inner: Mutex<wysiwyg::ComposerModel<Utf16String>>,
 }
 
+impl Default for ComposerModel {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
Actually to fix these I have a better/more clean suggestion... but it may take some time (not that much).
You can actually give as `derive` the `Default` trait. 
If you give the default trait as derive to all of the internal types of a type, then the type itself can also auto implement and derive default.

The idea is to give default to all of our types, so that we don't need to write any `impl` code to make Default work for that type

Reply:
Also while default is very cool, we should also use it, and change any instance of `Type::new()` with `Default::default` which is the generic function that will return the default impl

Reply:
I'll have a go, agree it would be cleaner, it does seem redundant having these blocks as I've added them

Reply:
@Velin92 What do you think to this? I've switched to trying to derive defaults where possible, the only place where I think it isn't clear what to do is in the DomNode, where I have had to implement it manually. I opted for a text node, felt it was a coin toss between that and a container node.

Reply:
i think it's good I am wondering if we really need then DomNode to implement default since maybe it should not have a default constructor?

Reply:
If DomNode doesn't implement a default, then it goes up the tree, so DomStruct can't derive a default, so ComposerState can't derive a default.

Maybe for ComposerState we stick to the implementation of default and just calling new()?

Reply:
Yeah I understand.
By checking DomStruct I see that default DomNode it may include is of Generic type, so maybe DomNode should be a generic container too as default value.
I guess that would make sense since it seems the most generic and defaulty of the options.

Reply:
Just noting for oversight over the Christmas-New year period:
This was switched over to use the container node as the default type, and the implementation was copied from dom_struct.rs to make sure the default was set up correctly